functions/public/Invoke-WPFMicrowin.ps1

```diff
@@ -238,6 +234,9 @@ public class PowerManagement {
         # Write-Host Error code $LASTEXITCODE
         Write-Host "Done disabling Teams"
 
+        Write-Host "Fix Windows Volume Mixer Issue"
+        reg add "HKLM\zDEFAULT\Microsoft\Internet Explorer\LowRegistry\Audio\PolicyConfig\PropertyStore" /f
```

Comment:
```suggestion
        reg add "HKLM\zNTUSER\Microsoft\Internet Explorer\LowRegistry\Audio\PolicyConfig\PropertyStore" /f
```

See the reason why in the Discord DM

Reply:
And the "software" does not have to be after zuser and microsoft?

Reply:
> And the "software" does not have to be after zuser and microsoft?

No. `zNTUSER`, which is basically the user's `NTUSER.DAT` file, contains the majority of HKCU (the SOFTWARE key).

So you're already loading the SOFTWARE hive